service/fixed/plans/plan_subscriptions/service.go

```diff
@@ -0,0 +1,44 @@
+package plan_subscriptions
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/RedisLabs/rediscloud-go-api/service/fixed/plans"
+)
+
+const root = "/fixed/plans/subscriptions"
+
+type Log interface {
+	Printf(format string, args ...interface{})
+}
+
+type HttpClient interface {
+	Get(ctx context.Context, name, path string, responseBody interface{}) error
+	GetWithQuery(ctx context.Context, name, path string, query url.Values, responseBody interface{}) error
+}
+
+type API struct {
+	client HttpClient
+	logger Log
+}
+
+func NewAPI(client HttpClient, logger Log) *API {
+	return &API{client: client, logger: logger}
+}
+
+// List will list all subscriptions with a specific plan
```

Comment:
```suggestion
// List will list all plans upgradable from a given subscription
```
Or words to that effect, I believe!